### PR TITLE
Moved distribution close to being "CPANTS clean"

### PR DESCRIPTION
**Pull request by neilb**
Hi Ali,

This has a number of minor changes to get the distribution following various CPAN conventions:
- Added a Changes file, which follows conventions in CPAN::Changes::Spec
- Added an abstract to the pod
- Specified min version of perl (based on perlver, which is in the Perl::MinimumVersion dist)
- Added a SEE ALSO section to the pod

The only thing CPANTS doesn't like now is the fact that the CPAN distribution doesn't contain a README. Is there a dzil plugin that will render your README.md as README, to include in the release?

Cheers,
Neil

### CODE REVIEW

**Comment by ironcamel** [comment]
Why does the minimum perl version need to be changed to 5.12?

**Comment by neilb** [comment]
perlver --blame on the lead module says that use of ... requires 5.12.0
I'm not familiar with Method::Signatures though - is its use of ... something else?

**Comment by ironcamel** [comment]
@neilbowers yeah, https://metacpan.org/pod/Method::Signatures#The-yada-yada-marker provides the `...` thing

**Comment by neilb** [comment]
Ah, in that case the min version should probably be (at least) 5.8.1, since Method::Signatures requires that. I haven't checked the other dependencies.

**Comment by aanari** [comment]
@neilbowers Can we please modify min version to a 5.8 version?  Otherwise I appreciate your efforts here and would like to merge this PR.  Thanks!

**Comment by neilb** [comment]
done!

File: cpanfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ requires "Throwable::Error" => "0";
 requires "WebService::BaseClientRole" => "0.0004";
 requires "WebService::Client" => "0";
 requires "aliased" => "0";
+requires "perl" => "5.012";
```
Comment (aanari) [inline cpanfile]: I'm still seeing this for some reason even though you added `use 5.008001` in Lob.pm, hrmm...

**Comment by neilb** [comment]
I think building the dist for release will update the cpanfile, and take the perl version required from the code, but I pushed an update to the cpanfile anyway.

**Comment by aanari** [comment]
Great, thanks for your help @neilbowers! +1

**Pull request merged**